DAO/src/erc20_functions.rs

```diff
@@ -0,0 +1,76 @@
+use gstd::{exec, msg, prelude::*, ActorId};
+use primitive_types::H256;
+const GAS_RESERVE: u64 = 500_000_000;
+
+pub async fn transfer_tokens(token_id: &ActorId, from: &ActorId, to: &ActorId, amount: u128) {
+    let transfer_data = TransferData {
+        from: H256::from_slice(from.as_ref()),
+        to: H256::from_slice(to.as_ref()),
+        amount,
+    };
+
+    let transfer_response: TokenEvent = msg::send_and_wait_for_reply(
+        *token_id,
+        TokenAction::Transfer(transfer_data),
+        exec::gas_available() - GAS_RESERVE,
+        0,
+    )
+    .await
+    .unwrap();
```

Comment:
Usually we use `expect()` with explanation instead of `unwrap()`. 
```suggestion
    .expect("Send error when transfer tokens");
```